.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 name: build
 on:
     push:
-        branches: [main]
+        branches: [main, feature-Legion]
```

Comment:
will this publish the package to npm?

Reply:
Nope. It just runs the build. There is a different YML file for release.